api/api/error_handler.py

```diff
@@ -221,7 +221,7 @@ async def problem_error_handler(request: ConnexionRequest, exc: exceptions.Probl
         del problem['detail']
 
     return json_response(data=problem, pretty=request.query_params.get('pretty', 'false') == 'true',
-                         status_code=exc.__dict__['status'], content_type=ERROR_CONTENT_TYPE)
+                         status_code=exc.__dict__['status'])
```

Comment:
This was also returning problem+json:
![image](https://github.com/wazuh/wazuh/assets/23361101/ef932010-28ba-4612-be28-161de0c1cb3f)